CLAUDE.md

```
@@ -0,0 +1,119 @@
+# Odyssey — Khoury College Education Platform
+
+Next.js 15 + Strapi 4.22 education platform at khouryodyssey.org. Makes education content discoverable and completable by Northeastern students through bite-sized learning units called "droplets."
+
+## Project Map
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)